server/src/controllers/game.controller.js

```diff
@@ -100,6 +100,27 @@ class GameController {
       throw new CustomError(`Unable to get all Games: ${error}`, 500);
     }
   }
+  // Fetch a single game
+  async getById(req, res) {
+    try {
+      // request an info from the user
+      const game_id = req.params.id;
+
+      // Get all games from the database
+      const fetchedGame = await GameRepo.fetchOne(game_id);
+
+      // Return all games
+      if (fetchedGame.data !== null) {
+        res
+          .status(200)
+          .send(response("Games retrieved successfully", fetchedGame.data));
```

Comment:
‘data’ is an array you need to modify the check and pick the only item in the array if it is there